### DISCUSSION TITLE
Cloned part of your work

### DISCUSSION THREAD

**Discussion by patrikha** [johann-petrak/licenseheaders#67] [Q&A] [answered]
Hi, just want to mention that I took some of your ideas and extended it into a new application.
[copywrite](https://github.com/patrikha/copywrite)

please let me know if you want to be mentioned in the LICENSE or README files.

**Comment by johann-petrak** [comment, marked as answer]
Hi @patrikha  there is no obligation to do that according to the license, I did also not look how many of those ideas went into your software, so please feel free to decide on this however you see fit!
